templates/corporate/support/sales_support_request.html

```diff
@@ -0,0 +1,72 @@
+{% extends "zerver/portico_signup.html" %}
+
+{% set PAGE_TITLE = "Contact sales | Zulip" %}
+
+{% block portico_content %}
+<div class="register-account flex full-page">
+    <div class="center-block new-style">
+        <div class="pitch">
+            <h1>Contact sales</h1>
+        </div>
+
+        <form method="post" class="white-box" id="registration">
+            {{ csrf_input }}
+
+            <fieldset class="support-request">
+                <div class="input-box support-form-field">
+                    <label for="contact_sales_from" class="inline-block label-title">From</label>
+                    <div id="contact_sales_from" class="not-editable-realm-field">
+                        {{ user_email }}
+                        <br />
+                        Your message will be sent to
+                        <a href=mail:sales@zulip.com>Zulip Support</a>.
+                    </div>
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_role" class="inline-block label-title">Your role</label>
+                    <input id="sales_support_request_form_role" class="required" type="text" name="role" maxlength="{{ MAX_INPUT_LENGTH }}" required />
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_organization_name" class="inline-block label-title">Organization name</label>
+                    <input id="sales_support_request_form_organization_name" class="required" type="text" name="organization_name" maxlength="{{ MAX_INPUT_LENGTH }}" required />
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_organization_type" class="inline-block label-title">Organization type</label>
+                    <select id="sales_support_request_form_organization_type" name="organization_type" class="required">
+                        {% for org_type in ORG_TYPES %}
+                            <option
+                              value="{{ org_type.name }}"
+                              {% if org_type.id == realm_org_type %}
+                              selected
+                              {% endif %}
+                              >
+                                {{ org_type.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_organization_website" class="inline-block label-title">Organization website</label>
+                    <input id="sales_support_request_form_organization_website" class="required" type="url" name="organization_website" required />
```

Comment:
This validation seems to require the `https` part; is there a way we can turn that off?

Reply:
![Screenshot 2024-10-01 at 3 50 08 PM](https://github.com/user-attachments/assets/79779fd3-1438-479c-a56e-ababb84dbb0a)

Submitting URL with `http` also works for me.

Reply:
Sure, but like a lot of people probably will type `foo.com`.

This can be a follow-up, since it's present in the other forms already.